components/constants/brave_switches.h

```diff
@@ -52,6 +52,11 @@ inline constexpr char kTor[] = "tor";
 // Override update feed url. Only valid on macOS.
 inline constexpr char kUpdateFeedURL[] = "update-feed-url";
 
+// Don't set kShowAlways for non-stable channel.
+// It's useful to test SidebarShowAlwaysOnStable w/o griffin.
+inline constexpr char kDontShowAlwaysSidebarOnNonStable[] =
+    "dont-show-always-on-sidebar-non-stable";
```

Comment:
optional: maybe simply

```suggestion
inline constexpr char kDontShowSidebarOnNonStable[] =
    "dont-show-on-sidebar-non-stable";
```

Reply:
Simplified :)